### PR TITLE
Create project structure of the app using Angular

### PR DESCRIPTION
**Pull request by kdaglas**
### What does this PR do?
 This PR is to set up the nomaCakes UI project from scratch using angular CLI

### Description of Task to be completed?
 Use Angular 8 and set up the project and create the necessary folders required for the components

### How should this be manually tested?
- Fetch this branch by running the command `git fetch origin ch-setup-project` in the terminal
- Check into this branch by running `git checkout ch-setup-project` in the terminal
- Start the application by running `npm start`

### CODE REVIEW

**Comment by kdaglas** [comment]
[Setup Project from scratch](https://trello.com/c/fa8jgi3d/39-setup-project-from-scratch)

File: tslint.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    "no-outputs-metadata-property": true,
+    "template-banana-in-box": true,
+    "template-no-negated-async": true,
+    "use-lifecycle-interface": true,
+    "use-pipe-transform-interface": true
+  },
+  "rulesDirectory": [
+    "codelyzer"
+  ]
+}
```
Comment (kizzanaome) [inline tslint.json]: Please add a new line at the end.

**Review by kizzanaome — APPROVED** [review]

File: e2e/protractor.conf.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    defaultTimeoutInterval: 30000,
+    print: function() {}
+  },
+  onPrepare() {
+    require('ts-node').register({
+      project: require('path').join(__dirname, './tsconfig.json')
+    });
+    jasmine.getEnv().addReporter(new SpecReporter({ spec: { displayStacktrace: true } }));
+  }
+};
```
Comment (kizzanaome) [inline e2e/protractor.conf.js]: Please add a new line.

File: browserslist [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+# https://github.com/browserslist/browserslist#queries
+
+# You can see what browsers were selected by your queries by running:
+#   npx browserslist
+
+> 0.5%
+last 2 versions
+Firefox ESR
+not dead
+not IE 9-11 # For IE 9-11 support, remove 'not'.
```
Comment (kizzanaome) [inline browserslist]: Please add a new line.

File: angular.json [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+          "configurations": {
+            "production": {
+              "devServerTarget": "nomaCakes-UI:serve:production"
+            }
+          }
+        }
+      }
+    }},
+  "defaultProject": "nomaCakes-UI"
+}
```
Comment (kizzanaome) [inline angular.json]: Please add a new line.

**Pull request merged**